nss-day-cohort-60/national-parks-django-api
# Sc events more updates

Added 'end_date' and more events into sql data
Added 'event_type' table that is embedded into events serializer

To Test:

Switch to sc-events-more-updates

make migrations in your database 

insert SQL data into database 

load event_type fixture into database  -- python3 manage.py loaddata event_types


create a GET request in POSTMAN to http://localhost:8000/events, 


see 'end_date' in post body
see 'event_type' in post body
